link-config.json

```
@@ -1,5 +1,5 @@
 {
-  "aliveStatusCodes": [200, 206, 402],
+  "aliveStatusCodes": [200, 206, 400, 402, 403],
```

```suggestion
  "aliveStatusCodes": [200, 206, 402],
```

In most of the cases 400 and 403 is a sign for bad link and we don't want to mark those codes as OK. I understand that you added this due to the few links that the build failed on, but I have explained in detail why those links failed here: https://github.com/cncf/tag-env-sustainability/pull/289#issuecomment-1828103998

---

You may ignore the link check build error as part of your PR - it will be fixed once #289 is merged.